tests/vmi_networking_test.go

```diff
@@ -607,7 +607,7 @@ var _ = Describe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][le
 			return vmi
 		}
 
-		table.DescribeTable("[test_id:1780][label:masquerade_binding_connectivity]should allow regular network connection", func(ports []v1.Port, withCustomCIDR bool) {
+		table.DescribeTable("[Conformance][test_id:1780][label:masquerade_binding_connectivity]should allow regular network connection", func(ports []v1.Port, withCustomCIDR bool) {
```

Comment:
make sense to also check bridge binding ?

Reply:
We consider bridge binding unstable and unsupportable, so I omitted it. We had issues with the binding in the past and I since it heavily changes the pod networking, it may be problematic for some CNIs. We are looking for a replacement of the binding before we go v1 with KubeVirt.

Reply:
Fair enough